### DISCUSSION TITLE
Lite version for reading JSON files only?

### DISCUSSION THREAD

**Discussion by doughunsaker** [nlohmann/json#4201] [General]
I'm writing a book about flight simulation and would like to include some example code. I like to read in all the parameters for the simulator in JSON format. However, I'm fairly certain this only uses a small fraction of what the .hpp file is capable of. Is there a "lite" version of this code that only reads in JSON data that would be appropriate to include in the book? I would give full credit of course to the source.

Thanks!

**Comment by jwalkerbg** [comment]
Probably the code that is involved in reading is used to generate code for executable file only. Other code, that is not referred does not generate code.
